webapp/src/Controller/Jury/ClarificationController.php

```diff
@@ -352,7 +352,13 @@ public function changeQueueAction(Request $request, int $clarId): Response
         if ($queue === "") {
             $queue = null;
         }
-        $clarification->setQueue($queue);
+
+        // Either this is the first in a thread (of 1 or more) or this 2nd or more.
```

Comment:
This comment is not really clear to me: `or this 2nd or more.`

Reply:
Removed the comment as Nicky his fix made this easy enough to read in code.